amgut/lib/locale_data/american_gut.py

```diff
@@ -291,85 +291,258 @@
-    'TEXT_QUESTIONS_TITLE': 'Contacts and Questions',
-    'TEXT_QUESTIONS_DESCRIPTION_1': 'For questions, concerns, or complaints about this study, call please call Rob Knight at 303-492-1984 or email <a href="mailto:Rob.Knight@colorado.edu">Rob Knight</a>.',
-    'TEXT_QUESTIONS_DESCRIPTION_2': 'If you are injured as a result of participating in this study or for questions about a study-related injury, call Please call Rob Knight at 303-492-1984 or email <a href="mailto:Rob.Knight@colorado.edu">Rob Knight</a>.',
-    'TEXT_QUESTIONS_DESCRIPTION_3': 'If you have questions about your rights as a research study participant, you can call the Institutional Review Board (IRB). The IRB is independent from the research team. You can contact the IRB if you have concerns or complaints that you do not want to talk to the study team about. The IRB phone number is (303) 735-3702.',
-    'TEXT_I_HAVE_READ_1': 'I have read (or someone has read to me) this form. I am aware that I am being asked to be in a research study. I have had a chance to ask all the questions I have at this time. I have had my questions answered in a way that is clear. I voluntarily agree to be in this study.',
-    'TEXT_I_HAVE_READ_2': 'I am not giving up any legal rights by signing this form. I will be sent a copy of this form to the email address I used to sign up for the study.',
+    'AGE_0_6': '0-6 years',
+    'AGE_7_12': '7-12 years',
+    'AGE_13_17': '13-17 years',
+    'AGE_18': '18+ years',
+    'ASSENT_7_12': '''<p align='center'>University of California, San Diego<br/>
+Assent to Act as a Research Subject<br/>
+(Ages 7-12 years)</p>
+
+<p align='center'><b>Human bugs: why and where they live on you</b></p>
+
+
+<p>Dr Rob Knight and his research team are doing a research study to find out more about the trillions of  tiny living things likes bacteria that live in you or on you. You are being asked if you want to be in this study because you are different (in a good way) from everybody else and they are different from each other.</p>
+<p>If you decide you want to be in this research study, this is what will happen to you:</p>
+<p>We will ask you or your mom or dad to sample some place on your body (like skin or mouth) or your poop (from toilet paper) with something that looks like 2 Q-tips.</p>
+<p>Sometimes kids don't feel good while being in this study. You might feel a little bit sore where your skin is rubbed with the Q-tip.  Most people don't feel this.</p>
+<p>If you feel any of these things, or other things, be sure to tell your mom or dad.</p>
+<p>You don't have to be in this research study if you don't want to. Nobody will be mad at you if you say no. Even if you say yes now and change your mind after you start doing this study, you can stop and no one will be mad.</p>
+<p>Be sure to ask Dr. Knight or his research team to tell you more about anything you don't understand.</p>
+''',
+    'ASSENT_13_17':
```

Comment:
Why is this `ASSENT_13_17` whereas below it is `CONSENT_18`? Is this just the wording that the IRB approved, so we are going with it?